mantidproject/mantid
# Improving fitfunctions in python

Description of work.

To provide python objects for fit functions rather than strings as of present as described in
https://github.com/mantidproject/documents/blob/master/Design/Fit%20functions%20in%20python.md
Substantial work has been done.

**To test:**

Check that the unit tests for FunctionWrapper and simpleapi work. Read the FunctionWrapper unit test to see what work has been done. Try it out in your own scripts.

Fixes #19010. 

**Release Notes** 

*To do.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
